testsuite/features/upload_files/wait-end-of-cleanup-pxeboot.exp

```diff
@@ -1,17 +1,12 @@
 set pxeboot_ipv4 [lindex $argv 0]
+set timeout 20
 
 spawn /usr/bin/ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null $pxeboot_ipv4
 match_max 100000
 expect "*?assword:*"
 send -- "linux\r"
 expect "#"
-send -- "systemctl stop salt-minion\r"
-expect "#"
-send -- "systemctl disable salt-minion\r"
-expect "#"
-send -- "rm /etc/salt/minion.d/*.conf\r"
-expect "#"
-send -- "rm /etc/salt/minion_id\r"
+send -- "while test `systemctl is-active salt-minion` == 'active'; do sleep 1; done\r"
```

Comment:
Interesting. I did not know we had such a file.

Reply:
We have a few `expect` files (with `.exp` at the end).

They are usually used with PXE boot minion, where's no direct SSH connection and sometimes not even an IPv4 address.